keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/BaseImageCache.kt

```diff
@@ -10,7 +10,7 @@ interface BaseImageCache {
    * image, if it exists.
    * @throws UnknownBaseImage if there is no known base image for [os] and [label].
    */
-  fun getBaseAmiVersion(os: String, label: BaseLabel): String
+  fun getBaseAmiName(os: String, label: BaseLabel): String
 
-  val allVersions: Map<String, Map<String, String>>
+  val allVersions: Map<String, Map<BaseLabel, String>>
```

Comment:
This is just used by the Keel endpoint that lists out the base AMIs. Using the enum helped me keep it straight in my head